docs/examples/MicroXSReader.rst

```diff
@@ -352,46 +360,48 @@ data is stored. If none of these settings are modified, the default is
 to store all the data from the output file.
 
 .. code:: 
+    
+    >>> rc['microxs.getFY'] = False # True/False only
+    >>> rc['microxs.getXS'] = True # True/False only
+    >>> rc['microxs.getFlx'] = True # True/False only
 
-    rc['microxs.getFY'] = False # True/False only
-    rc['microxs.getXS'] = True # True/False only
-    rc['microxs.getFlx'] = True # True/False only
-
-:ref:`microxs-getFY`: True or False, store fission yields
+``microxs.getFY``: True or False, store fission yields
```

Comment:
Can you use the `:ref:` directives to link these to their locations in `docs/defaultSettings.rst`

Reply:
Modified.